docs/pai-management/doc/customized-deploy.md

```diff
@@ -0,0 +1,83 @@
+<!--
+  Copyright (c) Microsoft Corporation
+  All rights reserved.
+
+  MIT License
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+  BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+
+## Customized deploy 
+
+### Index
+- [Step 1. Prepare configuration](#c-step-1)
+- [Step 2. Prepare Deployment Environment](#c-step-2)
```

Comment:
"Step 2. Prepare Deployment Environment" should be step1 and 
"[Step 1. Prepare configuration](#c-step-1" should be step2 ?

as user should deploy dev-box then customize config inner dev-box?

Reply:
You could find the environment setup option in the configuration generation document

Reply:
so if you think this is the previous steps at below position, why also make this as a step2
![image](https://user-images.githubusercontent.com/5576848/49782832-ac05b180-fd52-11e8-907c-044edb4b34e9.png)

Reply:
Not every time, admin should write the configuration. Only the first time. But environment is necessary.

Reply:
As your config is doing inner your Deployment Environment. Configuration (step1) has dependency for development env.  Follow your current deployment flow: if user did step1 B. Write your own Cluster Configuration, ->Step 2. Prepare Deployment Environment. User will lost. As step1 B not mentioned configuration inner development env.